fuzz/fuzz_packfile_raw.c

```diff
@@ -0,0 +1,87 @@
+/*
+ * libgit2 raw packfile fuzz target.
+ *
+ * Copyright (C) the libgit2 contributors. All rights reserved.
+ *
+ * This file is part of libgit2, distributed under the GNU GPL v2 with
+ * a Linking Exception. For full terms see the included COPYING file.
+ */
+
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdio.h>
+
+#include <openssl/sha.h>
+
+#include "git2.h"
+
+#define REPOSITORY_PATH "fuzz.git"
+
+static git_repository *repo = NULL;
+static git_odb *odb = NULL;
+
+int LLVMFuzzerInitialize(__attribute__((unused)) int *argc,
+			 __attribute__((unused)) char ***argv)
```

Comment:
Please use the `GIT_UNUSED` macro instead of `__attribute((unused))`.

Reply:
Done